src/towncrier/newsfragments/316.misc.rst

```diff
@@ -0,0 +1 @@
+
```

Comment:
Maybe add a note that 3.8 and 3.9 are "officially" supported.

Like:

CPython 3.8 and 3.9 are now part of our automated test matrix and are officially supported.

Reply:
I would have thought we already had a newsfragment from when the matrix was expanded and this was forgotten...  but I just looked and didn't see one, I'll add one.